staging/src/k8s.io/client-go/rest/request_test.go

```diff
@@ -2867,8 +2877,8 @@ func testRetryWithRateLimiterBackoffAndMetrics(t *testing.T, key string, doFunc
 
 		// it's a success, so do the following:
 		//  - call metrics and update backoff parameters
-		"RequestResult.Increment",
 		"BackoffManager.UpdateBackoff",
+		"RequestResult.Increment",
```

Comment:
should we count a success as a retry?
If I send 4 and the last one succeed, I've only retried 3 times

Reply:
the final attempt (the result of which we return to the user) is counted toward "RequestResult.Increment", and all previous attempts are counted to "RequestRetry.IncrementRetry",

Reply:
the one you are pointing to (line 1881 - "RequestResult.Increment") is the final attempt we return to the caller - it's the final attempt (either a success or a failure)

Reply:
oh, sorry, I got confused by the name, then is ok